### PR TITLE
Table styling

### PR DESCRIPTION
**Pull request by dhaneshnm**

### CODE REVIEW

File: src/shared/hooks/useDataCart.ts [hunk 1]
```diff
@@ -66,13 +65,20 @@ const useDataCart = () => {
...
+      const item = await storage.getItem(inputResources[i]._self);
+      if (!item) {
+        uniqueResources.push(inputResources[i]);
+        await storage.setItem(inputResources[i]._self, inputResources[i]);
+      }
+    }
+
+    setResources([...resources, ...uniqueResources]);
+    notification.success({
+      message: 'Selected resources has been added to your data cart.',
```
Comment (Stafeeva) [inline src/shared/hooks/useDataCart.ts:81]:
```suggestion
      message: 'Selected resources have been added to your data cart.',
```

File: src/shared/containers/ResourceViewContainer.tsx [hunk 2]
```diff
@@ -31,6 +31,9 @@ const ResourceViewContainer: React.FunctionComponent<{
     }> | null
   ) => React.ReactElement | null;
 }> = ({ render }) => {
+  const x = useParams();
+
+  // @ts-ignore
```
Comment (Stafeeva) [inline src/shared/containers/ResourceViewContainer.tsx:36]: are we leaving `ts-ignore` here?

File: src/shared/hooks/useMeasure.ts [hunk 3]
```diff
@@ -16,8 +16,10 @@ export default function useMeasure() {
     width: 0,
     height: 0,
   });
+
   const [ro] = React.useState(
-    new ResizeObserver(([entry]) => set(entry.contentRect))
+    // @ts-ignore
```
Comment (Stafeeva) [inline src/shared/hooks/useMeasure.ts:21]: are we leaving `ts-ignore` here?

File: src/shared/views/GalleryView.tsx [hunk 4]
```diff
@@ -72,9 +72,10 @@ const GalleryView: React.FC = () => {
             <Modal
               visible={true}
               footer={null}
-              onCancel={() =>
-                history.push(`${background.pathname}${background.search}`, {})
-              }
+              onCancel={() => {
+                // @ts-ignore
```
Comment (Stafeeva) [inline src/shared/views/GalleryView.tsx:76]: 'ts-ignore` 😢

File: src/shared/views/SubAppsView.tsx [hunk 5]
```diff
@@ -20,6 +20,7 @@ const SubAppsView: React.FC<{
   const loginUrl = `/login?destination=${location.pathname.replace('/', '')}`;
 
   return (
+    // @ts-ignore
```
Comment (Stafeeva) [inline src/shared/views/SubAppsView.tsx:23]: 'ts-ignore` 😢

File: src/subapps/projects/components/TableComponent.tsx [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+const TableComponent: React.FC<{}> = () => {
```
Comment (Stafeeva) [inline src/subapps/projects/components/TableComponent.tsx:3]: is it used anywhere? maybe we don't need it after all

File: src/subapps/studioLegacy/views/StudioAdminView.tsx [hunk 7]
```diff
@@ -4,11 +4,17 @@ import StudioListContainer from '../containers/StudioListContainer';
 import { Link } from 'react-router-dom';
 
 const StudioAdminView: React.FunctionComponent<{}> = () => {
+  // @ts-ignore
```
Comment (Stafeeva) [inline src/subapps/studioLegacy/views/StudioAdminView.tsx:7]: 'ts-ignore` 😢

Comment (dhaneshnm) [reply src/subapps/studioLegacy/views/StudioAdminView.tsx:7]: ts ignores are due to typescript version upgrade. This does not work with the latest typecsript.  We can remove them before we merge to main.

File: src/subapps/projects/components/TableComponent.tsx (hunk 6 above)
Comment (dhaneshnm) [reply src/subapps/projects/components/TableComponent.tsx:3]: I guess, no. Will remove.

**Review by Stafeeva — APPROVED** [review]

**Pull request merged**